python_modules/dagster/dagster_tests/asset_defs_tests/test_asset_key.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from dagster import AssetKey, SourceAsset, asset, load_assets_from_current_module
+from dagster._core.errors import DagsterInvalidDefinitionError
+
+# These tests pin the current behavior around getting forward slashes into asset keys
+# We may want to forbid this in the future
+
+
+def test_forward_slashes_in_keys_fail():
```

Comment:
This is kinda confusing because the title implies that forward slashes are not allowed ever but there are some cases where you _can_ get forward slashes into asset keys. I think you should include those here.

Reply:
These are here: https://github.com/dagster-io/dagster/pull/25742/files/e2f824b0d9aeb821e6b78e662d6fdff93838fe10#diff-7756d0c2a7b56b4417e43d69462144072a9757f0d65aa0f5b157a79eb61b9beeR52 

Do you know other methods for getting slashes into actual assets (not just `AssetKey`)?

Reply:
>Do you know other methods for getting slashes into actual assets (not just AssetKey)?

No, it was a comment on the title of the test. Pls just add an explanatory comment above the test that forward slashes don't always fail, in the current state if I was skimming I would have the impression they never work.